src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -109,13 +109,22 @@ public async Task Handle(SearchParameterDefinitionManagerInitialized notificatio
 
         public async Task UpdateSearchParameterStatusAsync(IReadOnlyCollection<string> searchParameterUris, SearchParameterStatus status)
         {
+            EnsureArg.IsNotNull(searchParameterUris);
+
+            if (searchParameterUris.Count == 0)
+            {
+                return;
+            }
+
             var searchParameterStatusList = new List<ResourceSearchParameterStatus>();
             var updated = new List<SearchParameterInfo>();
             var parameters = (await _searchParameterStatusDataStore.GetSearchParameterStatuses())
                 .ToDictionary(x => x.Uri);
 
             foreach (string uri in searchParameterUris)
             {
+                _logger.LogTrace("Setting the search parameter status of '{0}' to '{1}'", uri, status.ToString());
```

Comment:
>Setting the search parameter status of '{0}' to '{1} [](http://example.com/codeflow?start=34&length=52)

super nit.
I think you can write it that way:

`Setting the search parameter status of '{uri}' to '{newstatus}`

it's a bit more readable.

Reply:
Good call, that is also more consistent with our other trace messages. Updated in fe49a0a.